e2e-tests/csv-source/upload.hydra

```diff
@@ -41,11 +41,18 @@ With Class cc:CubeProject {
                     ```
                     A,B,C
                     1,2,test
+                    1,2,test
+                    1,2,test
+                    1,2,test
```

Comment:
Two lines were not enough for csv-sniff to do its magic (Error: No newline characters found in your file... at CSVSniffer.sniff (/app/node_modules/csv-sniffer/index.js:631:10))

Reply:
Curious. Maybe the case of failed sniffing could make for a mocha test to check that it does not fail and a resource is indeed added to the store (can use `TestResourceStore` + `to.matchSnapshot(this)`